convert2rhel/unit_tests/main_test.py

```diff
@@ -253,6 +258,7 @@ def test_main_traceback_before_action_completion(self, monkeypatch, caplog, tmp_
         assert finish_collection_mock.call_count == 1
         assert rollback_changes_mock.call_count == 1
         assert summary_as_json_mock.call_count == 0
+        assert report_to_txt_mock.call_count == 0
         print(caplog.records)
```

Comment:
Oh, I see we have a print left here by someone else, could you take care of removing it from here, @hosekadam?